functions/imagessr.js

```diff
@@ -22,11 +22,11 @@ app.get(['/in/embed/**', '/in/like/**'], async (req, res) => {
       html: data,
       quality: 100,
       type: 'jpeg',
-      selector: '#__layout > div',
+      selector: '#__layout > div > div',
```

Comment:
There are two divs for this selector, I suggest provide a dedicated class for this selector
<img width="727" alt="image" src="https://user-images.githubusercontent.com/17264716/224456642-3c8a95b9-f516-408d-8ac1-3561677780a6.png">